content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -34,6 +48,11 @@ export class _StartupOverlay extends React.PureComponent {
     }, 400);
   }
 
+  setFormFields(r) {
```

Comment:
Why a new function? Why not just set the state in the callback of the request when you have the response handy?